frontend/src/modules/scaffold/views/scaffoldListView.js

```diff
@@ -92,9 +94,42 @@ define([
     },
 
     cloneItem: function(event) {
+      var flatItem = Helpers.flattenNestedProperties(this.editor.value);
+      var itemValues = _.values(flatItem);
+      var parentAttributes = Origin.scaffold.getCurrentModel().attributes;
+      _.each(itemValues, function(item) {
```

Comment:
No requirement for Underscore here:
```suggestion
      itemValues.forEach(function(item) {
```